README.md

```diff
@@ -1,36 +1,43 @@
-An implementation of the Martini CG force field in OpenMM.
+# An implementation of the Martini coarse-grained force fields in OpenMM.
 
-This is alpha quality software. Use at your own risk. Testing
-on your specific system is recommend. See the `tests` directory
-for example scripts to compare gromacs and openmm energies and
-forces.
+This repository describes an implementation of the Martini 2 (Marrink, S. J. et al., J. Phys. Chem. B 2007, 111, 7812–7824; de Jong, D. H. et al., J. Chem. Theory Comput. 2013, 9, 687–697) and Martini 3 (Souza, P. C. T. et al., Nat. Methods 2021, 18, 382–388) force fields for coarse-grained simulation in the OpenMM package (Eastman, P. et al., PLoS Comput. Biol. 13(7)). It includes a Python parser to convert GROMACS Martini topology files (.top) to OpenMM topologies and systems. 
```

Comment:
Can we convert these into footnotes? I can't remember the exact syntax, but I think this is pretty easy with markdown.